package.json

```
@@ -12,14 +12,14 @@
     "postbuild": "npm run sass && npm run sass:icons",
     "dev": "webpack --node-env development --progress",
     "watch": "webpack --node-env development --progress --watch",
-    "lint": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo -n \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern",
-    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo -n \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern --fix",
+    "lint": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern",
+    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern --fix",
```

I am not sure the `-n` if even necessary: 
```suggestion
    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo $appdir; fi; done) core --no-error-on-unmatched-pattern --fix",
```